### PR TITLE
Add ID localization task configure volume storage

### PR DESCRIPTION
**Pull request by danninov**
Add ID localization for task/configure-volume-storage. Issue #22296

### CODE REVIEW

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    ```shell
+    kubectl delete pod redis
+    ```
+
+
+
+## {{% heading "whatsnext" %}}
+
+
+* Lihat [Volume](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volume-v1-core).
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:124]: Update the link!

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+* Lihat [Pod](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core).
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:126]: Update the link!

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+* Selain penyimpanan pada disk lokal yang di sediakan oleh `emptyDir`, Kubernetes
+juga mendukung solusi penyimpanan _network-attached_, termasuk PD pada
+GCE dan EBS dari EC2, yang cenderung lebih disukai untuk data sangat penting dan akan menangani urusan detil seperti _mounting_ dan _unmounting_ perangkat pada node. Lihat
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:130]:
```suggestion
GCE dan EBS dari EC2, yang cenderung lebih disukai untuk data sangat penting dan akan menangani urusan detil seperti _mounting_ dan _unmounting_ perangkat pada Node. Lihat
```

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+[Volumes](/docs/concepts/storage/volumes/) untuk informasi detil.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:131]: Link to ID Docs. It is available.
```suggestion
[Volume](/id/docs/concepts/storage/volumes/) untuk informasi detil.
```

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
+---
+title: Mengatur Pod untuk Menggunakan Volume sebagai Penyimpanan
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:2]:
```suggestion
title: Mengatur Pod untuk Menggunakan Volume sebagai Tempat Penyimpanan
```

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+Laman ini menjelaskan bagaimana cara mengatur sebuah Pod untuk menggunakan Volume sebagai penyimpanan.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:9]:
```suggestion
Laman ini menjelaskan bagaimana cara mengatur sebuah Pod untuk menggunakan Volume sebagai tempat penyimpanan.
```

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Mengatur volume untuk Pod
+
+Pada latihan ini, kamu membuat sebuah Pod yang menjalankan sebuah Container. Pod ini memiliki sebuah Volume dengan tipe [emptyDir](/docs/concepts/storage/volumes/#emptydir)
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:25]: Link to ID Docs. It is available.
```suggestion
Pada latihan ini, kamu membuat sebuah Pod yang menjalankan sebuah Container. Pod ini memiliki sebuah Volume dengan tipe [emptyDir](/id/docs/concepts/storage/volumes/#emptydir)
```

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+    redis     1/1       Running   0          13s
+    ```
+
+1. Pada terminal lain, buka _shell_ untuk masuk ke Container yang sedang berjalan:
+
+    ```shell
+    kubectl exec -it redis -- /bin/bash
+    ```
+
+1. Di dalam _shell_, pergi ke `/data/redis`, kemudiann buat sebuah berkas:
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:55]: Typo?
```suggestion
1. Di dalam _shell_, pergi ke `/data/redis`, kemudian buat sebuah berkas:
```

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+    ```shell
+    NAME      READY     STATUS     RESTARTS   AGE
+    redis     1/1       Running    0          13s
+    redis     0/1       Completed  0         6m
+    redis     1/1       Running    1         6m
+    ```
+
+Pada titik ini, Container telah berakhir dan dimuat ulang. Hal ini karena Pod Redis memiliki
+[restartPolicy](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core)
```
[the remark this answers is not in the document]
Comment (danninov) [reply content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:97]: The link is corrrect. We don't have ID localization for kubernetes-api yet

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md (hunk 1 above)
Comment (danninov) [reply content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:124]: The link is corrrect. We don't have ID localization for kubernetes-api yet

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:97]: ok!

File: content/id/docs/tasks/configure-pod-container/configure-volume-storage.md (hunk 1 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/configure-volume-storage.md:124]: ok!

**Comment by irvifa** [comment]
Nice 🎉 
/approve

**Pull request merged**